docs/source/api/apollo-gateway.mdx

```diff
@@ -135,6 +134,14 @@ example of using `ApolloGateway`, see [The gateway](/federation/gateway/).
     });
     ```
 
+  * `performServiceHealthChecks`: `boolean`
```

Comment:
Naming question, thoughts on the shorter option of `serviceHealthCheck` — also non-plural?

I'm curious if a future version of this might accept a function that defines specifically what those health-checks do, or possibly an object with additional parameters.  Some imprecision in the name (removing "perform") might facilitate that?

Reply:
Good with it! 416fc3e